guardrails/cli/server/hub_client.py

```diff
@@ -64,7 +63,7 @@ def fetch_module_manifest(
 
 def fetch_module(module_name: str) -> ModuleManifest:
     creds = Credentials.from_rc_file(logger)
-    token = get_auth_token(creds)
+    token = creds.token
```

Comment:
Before we enforce auth on any validators in the hub we'll need to add a check if the token is expired or not and reprompt for a fresh token (also add refresh token support).  Will need to do this _now_ for quick submit